tool/plugin/lib/plugin.dart

```diff
@@ -523,7 +536,13 @@ class BuildCommand extends ProductCommand {
 
       separator('Building flutter-intellij.jar');
       await removeAll('build');
+      for (var file in spec.filesToSkip) {
+        await new File(file).rename('$file~');
+      }
       result = await runner.javac2(spec);
+      for (var file in spec.filesToSkip) {
```

Comment:
this block may want to be in a `finally` block, so that it's performed even if there are exceptions out of the compile stage

Reply:
Done.